app/src/main/java/com/android/sample/ui/workout/WorkoutCreationScreen.kt

```diff
@@ -390,3 +410,51 @@ fun WorkoutCreationScreen(
         })
   }
 }
+
+@Composable
+fun DateTimePicker(
+    selectedDateTime: LocalDateTime?,
+    onDateTimeSelected: (LocalDateTime) -> Unit,
+) {
```

Comment:
Could be placed in the composables package instead of in the WorkoutCreationScreen

Reply:
Done.